src/store/creator/__tests__/reducer-test.ts

```diff
@@ -106,6 +116,7 @@ describe('store/creator/reducer', () => {
             );
 
             expect(newState).toEqual({
+                color: '#111',
```

Comment:
This is a little confusing. Shouldn't this be set back to #000, since that's the default mock value?

Reply:
No this is intentional. I want to test that the color is not reset when toggle different modes.